docs/examples/styles/border_subtitle_align.py

```diff
@@ -0,0 +1,20 @@
+from textual.app import App
+from textual.widgets import Label
+
+
+class BorderSubtitleAlignApp(App):
+    def compose(self):
+        lbl = Label("My border is on the left.", id="label1")
```

Comment:
Suggest "My **subtitle** ..."

Reply:
Right. I don't know why the heck I thought "border" was a good word, there.